docs/meshstack.mesh-policies.md

```diff
@@ -0,0 +1,38 @@
+---
+id: meshstack.mesh-policies
+title: Mesh Policies
+---
+
+meshPolicies can be set up between meshObjects, i.e., meshCustomer, meshProject, meshLandingZone and
+meshCustomerUserGroup. In the administration area, there is an option of 'Policies' in the 'Compliance' section of the
+navigation area. After clicking policies, you can view all policies in the system. Since this is a BETA version we only
+support tags for policies between meshObjects.
+
+## Create Policies
```

Comment:
Please add information about the policies being applied for every object instance of the specified objects. This can cause problems so they better be sure

Reply:
Added at the end of 'Create Policies' section